test_scripts/Defects/8_2/3895_SDL_sends_redundant_BC_PolicyUpdate.lua

```diff
@@ -0,0 +1,107 @@
+--------------------------------------------------------------------------------------------------
```

Comment:
This test passes on the latest sdl_core develop (ran the test 500 times just to make sure it passed consistently) so I don't think this test will allow us to verify that the core PR fixes the issue.